src/predicates.ts

```diff
@@ -65,6 +65,6 @@ export const isRackBarcode: PredicateFn = function isRackBarcode(
 
 export const isNotNullish: PredicateFn = function isNotNullish<T>(
   value: T | null | undefined,
```

Comment:
```suggestion
  value: T,
```

Does Typescript like that?

Reply:
Good suggestion!